consent-ws/src/test/resources/sql/insert.sql

```diff
@@ -12,6 +12,7 @@ insert into dacuser(dacUserId,email,displayName,createDate) values(2,'test2@broa
 insert into dacuser(dacUserId,email,displayName,createDate) values(3,'test3@broad.com','testUser2','2015-08-05 13:58:50');
 insert into dacuser(dacUserId,email,displayName,createDate) values(4,'test4@broad.com','testUser3','2015-08-05 13:58:50');
 insert into dacuser(dacUserId,email,displayName,createDate) values(5,'oauthuser@broadinstitute.org','oauth','2015-08-05 13:58:50');
+insert into dacuser(dacUserId,email,displayName,createDate) values(666,'diegogil@broadinstitute.org','oauth','2015-08-05 13:58:50');
```

Comment:
Does this really need to be Diego's email? I think it should be a fake value like the previous values.